src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/EventListener.java

```diff
@@ -164,25 +166,39 @@ protected void schedule(GerritTrigger t, GerritCause cause, GerritTriggeredEvent
      * @param event   the event.
      * @param project the project to build.
      */
-    protected void schedule(GerritTrigger t, GerritCause cause, GerritTriggeredEvent event, AbstractProject project) {
+    protected void schedule(GerritTrigger t, GerritCause cause, GerritTriggeredEvent event, Job project) {
         BadgeAction badgeAction = new BadgeAction(event);
         //during low traffic we still don't want to spam Gerrit, 3 is a nice number, isn't it?
         int projectbuildDelay = t.getBuildScheduleDelay();
         if (cause instanceof GerritUserCause) {
             // it's a manual trigger, no need for a quiet period
             projectbuildDelay = 0;
-        } else if (project.getHasCustomQuietPeriod()
-                && project.getQuietPeriod() > projectbuildDelay) {
-            projectbuildDelay = project.getQuietPeriod();
+        } else if (project instanceof AbstractProject) {
```

Comment:
I personally don't think that's an issue.
On Jul 10, 2015 9:42 AM, "Tom Fennelly" <notifications@github.com> wrote:

> In
> src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/EventListener.java
> <https://github.com/jenkinsci/gerrit-trigger-plugin/pull/240#discussion_r34372420>
> :
>
> >          BadgeAction badgeAction = new BadgeAction(event);
> >          //during low traffic we still don't want to spam Gerrit, 3 is a nice number, isn't it?
> >          int projectbuildDelay = t.getBuildScheduleDelay();
> >          if (cause instanceof GerritUserCause) {
> >              // it's a manual trigger, no need for a quiet period
> >              projectbuildDelay = 0;
> > -        } else if (project.getHasCustomQuietPeriod()
> > -                && project.getQuietPeriod() > projectbuildDelay) {
> > -            projectbuildDelay = project.getQuietPeriod();
> > +        } else if (project instanceof AbstractProject) {
>
> Note getQuietPeriod defaults if not set.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jenkinsci/gerrit-trigger-plugin/pull/240/files#r34372420>
> .
>